<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonBarWidget</class>
 <widget class="QWidget" name="ButtonBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ButtonBarWidget</string>
  </property>
  <property name="windowOpacity">
   <double>0.300000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton {
    background-color: transparent;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin" stdset="0">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="deleteButton">
     <property name="toolTip">
      <string>Delete note</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="postnote.qrc">
       <normaloff>:/PostNote/Resources/trash-can.png</normaloff>:/PostNote/Resources/trash-can.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="hideButton">
     <property name="toolTip">
      <string>Hide note</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="postnote.qrc">
       <normaloff>:/PostNote/Resources/eye.png</normaloff>:/PostNote/Resources/eye.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>209</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="propertiesButton">
     <property name="toolTip">
      <string>Properties</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="postnote.qrc">
       <normaloff>:/PostNote/Resources/properties.png</normaloff>:/PostNote/Resources/properties.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="topicButton">
     <property name="toolTip">
      <string>Topics</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="postnote.qrc">
       <normaloff>:/PostNote/Resources/PostNote.ico</normaloff>:/PostNote/Resources/PostNote.ico</iconset>
     </property>
     <property name="popupMode">
      <enum>QToolButton::ToolButtonPopupMode::MenuButtonPopup</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="postnote.qrc"/>
 </resources>
 <connections/>
</ui>
